internal/rootcoord/root_coord_test.go

```diff
@@ -1822,9 +1839,9 @@ func TestRootCoord(t *testing.T) {
 	req2 := &milvuspb.CreateCollectionRequest{
 		Base: &commonpb.MsgBase{
 			MsgType:   commonpb.MsgType_CreateCollection,
-			MsgID:     3013,
-			Timestamp: 3013,
-			SourceID:  3013,
+			MsgID:     3014,
```

Comment:
Why change 3013 to 3014, 3014 to 3015, etc?

Reply:
To keep the Timestamp ordering intact in the UTs.